### ISSUE TITLE
BUG: RESTful API not compatible with OpenAI API

### ISSUE THREAD

**Issue by ZanePoe** [xorbitsai/inference#306]
### Describe the bug
you said your RESTful API(compatible with OpenAI API)，but i post a openai json ，it response a errror.
![image](https://github.com/xorbitsai/inference/assets/16047426/cc029855-fb5b-4358-a779-33d79912a1dd)

![image](https://github.com/xorbitsai/inference/assets/16047426/4b7dbde6-d481-4cde-a5bc-040ed2122b23)

my json payload is from openai  and change model name to model id:
```
{
     "model": "c0021658-3289-11ee-89d1-00155d722c2a",
     "messages": [{"role": "user", "content": "Say this is a test!"}],
     "temperature": 0.7
   }
```
![image](https://github.com/xorbitsai/inference/assets/16047426/36be3c3e-b0a3-467e-9a17-e2e6e3be0472)

the error info:
```
getattr(self._model, "chat")(prompt, *args, **kwargs)
TypeError: [address=127.0.0.1:32829, pid=116] ChatglmCppChatModel.chat() takes from 2 to 4 positional arguments but 5 were given
```

**Comment by UranusSeven** [comment]
Thanks for reporting this issue! @jiayini1119 Could you pls have a look at this issue?

**Comment by jiayini1119** [comment]
Hi, thank you for reporting the issue! The bug fix is provided in #309.
